### PR TITLE
add jupyter-starters-reinforcement-learning-dennybritz

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Notes:
- this is kind of weird one: 
  - it's basically a UI launcher
    -  and a **validated set of dependencies**
  - for an existing repo of notebooks... 
  - it's a bit dated...
    - or _ancient_ in ML terms, perhaps? 
  - ... but just about everything appears to work under test except for some deprecated tensorflow calls... 
    - but that's half the fun of ML, right?

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/staged-recipes @conda-forge/help-python ready for review!

File: recipes/jupyter-starters-reinforcement-learning-dennybritz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - pandas
+    - psutil
+    - python >=3.6
+    - tensorflow <2
+    - scikit-learn
+
+test:
+  requires:
+    - pip
+    - pytest
```
Comment (xhochy) [inline recipes/jupyter-starters-reinforcement-learning-dennybritz/meta.yaml:39]: This doesn't seem to be used.
```suggestion
```

File: recipes/jupyter-starters-reinforcement-learning-dennybritz/build_starter.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
+"""build a jupyter-starter
```
Comment (xhochy) [inline recipes/jupyter-starters-reinforcement-learning-dennybritz/build_starter.py:1]: Can you clarify on the license of these scripts? Have you written them for the recipe or do they come from upstream?

File: recipes/jupyter-starters-reinforcement-learning-dennybritz/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipes/jupyter-starters-reinforcement-learning-dennybritz/meta.yaml:39]: Used in `run_test.py`!

File: recipes/jupyter-starters-reinforcement-learning-dennybritz/build_starter.py (hunk 2 above)
Comment (bollwyvl) [reply recipes/jupyter-starters-reinforcement-learning-dennybritz/build_starter.py:1]: nah, i wrote'm here. If this ends up decent, I might try to get them into something grayskull adjacent. Will update.

Comment (bollwyvl) [reply recipes/jupyter-starters-reinforcement-learning-dennybritz/build_starter.py]: added conda-forge bsd to both

**Comment by bollwyvl** [comment]
I'll likely be building more of these, and expanding them with more features (e.g. annotations, anki cards, bibliography dumps, etc). Thinking about this more, it might be better as a split recipe of:

- a "source" package (which, _man_, I sure wish conda-build did for me _anyway_) without any modifications of the original repo that puts things in `share/github.com/{author}/{repo}`
- a jupyter-starters package that knows where that is, and puts things in `share/jupyter/starters`

However, I'm still stuck with what to _name_ all the things.

**Comment by bollwyvl** [comment]
@xhochy updated... lemme know if there's any other concerns!

**Review by xhochy — APPROVED** [review]

**Comment by bollwyvl** [comment]
:taco: thanks!

**Pull request merged**